scoresheets/SetTable.tex

```diff
@@ -3,10 +3,9 @@
 \begin{scorelist}
 	\scoreheading{Main Goal}
 	\scoreitem{1000}{Neatly arrange tableware and cutlery on the table (5 objects)}
-	\penaltyitem[6]{50}{Pointing at object}
-	\penaltyitem[6]{50}{Handover an object}
-	\penaltyitem[3]{200}{Bypassing tableware placement}
-	\penaltyitem[3]{250}{Bypassing cutlery placement}
+	\penaltyitem[5]{50}{Pointing at object}
+	\penaltyitem[5]{100}{Handover an object}
+	\penaltyitem[5]{200}{Bypassing placement}
```

Comment:
Handling cutlery is harder than normal objects.

Reply:
But a object is 200 points, so getting a reduction of more than 200 points is useless.